tests/integration/changed-yum-conf/main.fmf

```diff
@@ -0,0 +1,8 @@
+summary: changed-yum-conf
+
+/good:
+  tag+:
+    - good_test
```

Comment:
this is not needed, right?

Reply:
I think so as well: https://github.com/oamg/convert2rhel/pull/189#discussion_r668837673